.github/workflows/test.yml

```diff
@@ -4,21 +4,21 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
+      - uses: actions/checkout@v2
 
-    - uses: leafo/gh-actions-lua@v6
-      with:
-        luaVersion: "5.3.5"
+      - uses: leafo/gh-actions-lua@v8.0.0
+        with:
+          luaVersion: 5.4
```

Comment:
At least one sample solution depends upon `bit32` support which was removed in 5.4. I'm okay with moving to 5.4, but that exercise will need to be updated.

Reply:
Yepp, I saw it. I'll try to fix it. :)

Reply:
Done